scripts/build_attestation_agent.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -x
```

Comment:
You could add the standard Kata shell script "boiler plate" code here:

```suggestion
set -o errexit
set -o nounset
set -o pipefail

[ -n "$BASH_VERSION" ] && set -o errtrace
[ -n "${DEBUG:-}" ] && set -o xtrace
```

Reply:
good suggestion, will update this part.